File: app/src/main/java/padev/badhabits/application/mvp/presenter/habit/HabitPresenter.java
package padev.badhabits.application.mvp.presenter.habit;

import com.arellomobile.mvp.InjectViewState;
import com.arellomobile.mvp.MvpPresenter;

import padev.badhabits.application.mvp.view.IHabitView;

@InjectViewState
public class HabitPresenter extends MvpPresenter<IHabitView> implements IHabitPresenter {
}

File: app/src/main/java/padev/badhabits/application/mvp/presenter/habit/HabitPresenter.kt
package padev.badhabits.application.mvp.presenter.habit;

import padev.badhabits.application.mvp.model.interactor.habit.HabitInteractor
import padev.badhabits.application.mvp.view.IHabitView

class HabitPresenter(val viewState: IHabitView): IHabitPresenter {

    val interactor = HabitInteractor(this)


    override fun getHabitDetails(id: Long) {


    }
}
